src/index.js

```diff
@@ -25,7 +22,11 @@ module.exports = async function({
   }
   await page.setContent(html)
   const element = await page.$('body')
-  const buffer = await element.screenshot({ path: output, type, omitBackground: transparent, encoding })
+  const screenShotOptions = { type, omitBackground: transparent };
+  if (output) {
+    Object.assign(screenShotOptions, { path: output });
+  }
+  const buffer = await element.screenshot(screenShotOptions)
```

Comment:
The previous implementation was good. If output is `undefined` path will be undefined. What do you think of it?

Reply:
ah, i understand ur issue. I think assigning initial value fixes it, like `output = ''` . then, we can check if it is not empty @frinyvonnick

Reply:
I think we don't need to check it. `output` is `undefined` by default so :

```js
const buffer = await element.screenshot({ path: output, type, omitBackground: transparent, encoding })
```

will do the same as : 

```js
const screenShotOptions = { type, omitBackground: transparent };
if (output) {
  Object.assign(screenShotOptions, { path: output });
}
const buffer = await element.screenshot(screenShotOptions)
```

The scope of this PR is to make output optional. Removing this code is enough : 

```js
  if (!output) {	
    throw Error('You must provide an output property.')	
  }
```